keyboards/vitamins_included/rev2/config.h

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2019 Mikkel Jeppesen
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define SPLIT_HAND_PIN B4
+#define SOFT_SERIAL_PIN D0
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBEE5
+#define PRODUCT_ID      0xBEEB
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    Duckle29
+#define PRODUCT         Vitamins included
+#define DESCRIPTION     A toolless split keyboard for the cheapish makers
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+
+// wiring of each half
+#define MATRIX_ROW_PINS { F5, F6, C7, F7 }
+#define MATRIX_COL_PINS { F1, F4, E2, B6, D7, D6}
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_LEVELS 3
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING 5
```

Comment:
My fault; I gave you the wrong define name before.

```suggestion
#define DEBOUNCE 5
```